### ISSUE TITLE
Responsively gets stuck in touch mode very easily

### ISSUE THREAD

**Issue by swilkewitz** [responsively-org/responsively-app#573]
I get stuck in touch mode after only a few clicks. When I go to press the touch mode button at the top right it won't deactivate. I have to restart the app every time I get stuck

I noticed that my mouse icon is still set to touch mode even when I'm interacting with the Responsively app. Why is that the case? That seems odd to me.

In general, it seems like touch mode is an afterthought... shouldn't it be a first-class feature since the tool is meant to test mobile devices? I think touch mode should be the default for mobile devices.

Guess I'll keep shopping around.

---
Windows 10

**Comment by manojVivek** [comment]
@swilkewitz Yeah, this is a known behavior, but it should get deactivated when you click(tap) the icon again.

> I noticed that my mouse icon is still set to touch mode even when I'm interacting with the Responsively app. Why is that the case? That seems odd to me.

This is because of a disability in Electron, the touch mode cannot be activated specifically for a Webview, it will get activated for the whole app. 

We will keep this open to see if any of the future releases of Electron makes this possible.

**Comment by swilkewitz** [comment]
"but it should get deactivated when you click(tap) the icon again."

It doesn't, that's why i posted this bug
